### PR TITLE
More Cleanup and Pepping

### PR DESCRIPTION
**Pull request by wslippey**

### CODE REVIEW

**Comment by wslippey** [comment]
@ppfaWebtech

File: CloudFlare/cloudflare.py [hunk 1]
```diff
@@ -438,6 +488,7 @@ def __init__(self, email=None, token=None, certtoken=None, debug=False, raw=Fals
         self._base = self._v4base(email, token, certtoken, base_url, debug, raw)
 
         # add the API calls
-        api_v4(self)
+        api_v4(self)  # todo: This should really be part of the class?
```
Comment (ppfaWebtech) [inline CloudFlare/cloudflare.py]: @wslippey Unless we're trying to futureproof api_v5, etc... which seems strange with such a small footprint. And AFAIK CloudFlare doesn't run multiple API's, so for both those reasons this seems superfluous.

**Comment by mahtin** [comment]
WOW Thanks @wslippey and @ppfaWebtech ! This will take a little while to read over and accept. I see you were quite complete in your work. More later!

**Comment by wslippey** [comment]
@mahtin Apologies, I meant to make this PR against our fork. It's not ready and there are still issues to be addressed.

**Comment by mahtin** [comment]
@wslippey - take your time; I have some other items to deal with (unrelated) and feel free to contact me via my contact info.

**Pull request closed without merging**